xml/osc_building.xml

```diff
@@ -304,6 +306,68 @@ The buildroot was: /var/tmp/build-root/openSUSE_Tumbleweed-x86_64</screen>
   	   in obs://$PROJECT/$REPOSITORY style.
      </para>
    </listitem>
+   <listitem>
+     <para>A predefined milestone <command>OBS-Milestone: milestone</command> 
+     </para>
+     <para>This can predefine a milestone name (eg Beta1) which will be used in cases of a release operation. The candidate will receive it's final tag then.
+     </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+ <sect1>
+  <title>Docker Container Specials (&docker;)</title>
+   <!--
+    1. Create a repository in a project
+       - enable all architectures
+       - path elements are optional if defined via zypper ar commands
+    2. Create project config, setting repo to
+       - Type: docker
+       - BuildEngine: podman (optional)
+       - Repotype: none
+    3. Create a package
+    4. Submit Dockefile
+    
+   -->
+   <para>
+	OBS needs to parse RUN commands to detect build dependencies (repositories and packages).
+       Currently calls from zypper, apt-get, yum and dnf are supported.
+       Additional downloads need to be covered by source services. This ensures a reproducible
+       builds and a safe build environment without network access.
+   </para>
+
+   <para>
+     OBS extends Docker functionality with following options. These covers cases which would need explicit 
+     command line commands, so they are covered via the tags to have them in a reproducible way.
```

Comment:
I have a bit of a hard time grasping the original version, an attempt at clarifying:

```suggestion
     OBS extends Docker functionality with the below tags.
     Instead of these options, you would otherwise need explicit command-line commands.
     However, to make it possible to create reproducible builds, use the OBS tags.
```

I am not sure that wording adequately captures the idea though.